package.json

```diff
@@ -17,22 +17,32 @@
     "node": "10.16.3"
   },
   "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "^5.14.0",
+    "@fortawesome/react-fontawesome": "^0.1.12",
+    "@fortawesome/fontawesome-svg-core": "^1.2.32",
```

Comment:
typo?
fort/font

Reply:
@julienledem It seems that this is correct. The organization that ships `fontAwesome` is called `fortAwesome`. It's a bit confusing, but here is their GitHub organization page: 
https://github.com/FortAwesome